setup.py

```diff
@@ -70,6 +70,7 @@
     "typeguard==2.13.3",
     "fastapi>=0.68.0,<1",
     "uvicorn[standard]>=0.14.0,<1",
+    "gunicorn",
```

Comment:
you probably need to regenerate some requirements with this with:
- `make lock-python-ci-dependencies PYTHON={3.8,3.9,3.10}`
- `make lock-python-dependencies PYTHON={3.8,3.9,3.10}`

Reply:
@adchia Thanks for the advice.
When I ran the suggested command, the versions of other libraries were also changed, but only gunicorn was added first.